src/js/controller/model.js

```diff
@@ -271,7 +271,7 @@ define([
             this.set('mediaModel', this.mediaModel);
             this.set('position', item.starttime || 0);
             this.set('duration', (item.duration && utils.seconds(item.duration)) || 0);
-
+            this.set('minDvrWindow', item.minDvrWindow);
```

Comment:
Should this be set before duration, incase `item.duration != 0` and `change:duration` will determine `streamType`?